Libraries/Components/Button.windows.js

```diff
@@ -155,35 +155,46 @@ class Button extends React.Component {
     if (Platform.OS === "windows") {
       const tabIndex = this.props.tabIndex || 0;
       const windowsTabFocusable = !disabled && tabIndex >= 0;
-      content =
-        <FocusableWindows
-          ref={this._setFocusableRef}
+      return (
+        <Touchable
+          importantForAccessibility="no"
+          testID={testID}
           disabled={disabled}
-          isTabStop={windowsTabFocusable}
-          tabIndex={tabIndex}
-          disableSystemFocusVisuals={this.props.disableSystemFocusVisuals}
-          handledKeyDownKeys={DOWN_KEYCODES}
-          handledKeyUpKeys={UP_KEYCODES}
-          onKeyDown={this._onKeyDown}
-          onKeyUp={this._onKeyUp}
-          onFocus={this._onFocus}
-          onBlur={this._onBlur}
-        >
-          {content}
-      </FocusableWindows>;
+          onPress={onPress}>
+            <FocusableWindows
+              ref={this._setFocusableRef}
+              disabled={disabled}
+              isTabStop={windowsTabFocusable}
+              tabIndex={tabIndex}
+              disableSystemFocusVisuals={this.props.disableSystemFocusVisuals}
+              accessibilityComponentType="button"
+              accessibilityLabel={accessibilityLabel}
+              accessibilityTraits={accessibilityTraits}
```

Comment:
Why we do we have to move these from `Touchable` to `FocusableWindows`?

Reply:
The reason was that In Windows Narrator there is an option for Narrator to follow focus. When user moves focus to the button with this option "on", Narrator will try to get and use automation peer for FocusableWindows (I mean whatever native control it finally renders and passes its own accessibility* props to). If we leave acc* props only on Touchable, in this case Narrator will not read proper text.

Reply:
I added explanation in a comment.

---
In reply to: [158559738](https://github.com/reseul/react-native-windows/pull/1#discussion_r158559738) [](ancestors = 158559738)